trie/triedb/pathdb/journal.go

```diff
@@ -341,6 +343,14 @@ func (db *Database) Journal(root common.Hash) error {
 	if l == nil {
 		return fmt.Errorf("triedb layer [%#x] missing", root)
 	}
+	disk := db.tree.bottom()
```

Comment:
Seems only for logging or testing? If unnecessary, delete these lines.

Reply:
deleted